frontend/src/lib/api.ts

```diff
@@ -1232,8 +1233,25 @@ const api = {
         ): Promise<NotebookType> {
             return await new ApiRequest().notebook(notebookId).update({ data })
         },
-        async list(): Promise<PaginatedResponse<NotebookType>> {
-            return await new ApiRequest().notebooks().get()
+        async list(
+            contains?: { type: NotebookNodeType; attrs: Record<string, string> }[]
+        ): Promise<PaginatedResponse<NotebookType>> {
+            const apiRequest = new ApiRequest().notebooks()
+            if (!!contains?.length) {
+                const containsString = contains
+                    .map(({ type, attrs }) => {
+                        const target = type.replace(/^ph-/, '')
+                        if (target === 'recording') {
+                            return `${target}:${attrs['id']}`
+                        } else {
```

Comment:
See my other comment from the backend but - why not just remove the "if target"? Most of the types we have are keyed by id already so it would JustWork and then we can add the exception situations later?

Reply:
yep follow-up